.mdox.yaml

```
@@ -0,0 +1,165 @@
+version: 1
+
+inputDir: "docs"
+outputDir: "website/docs-pre-processed/tip"
+extraInputGlobs:
+  - "CHANGELOG.md"
+  - "SECURITY.md"
+  - "MAINTAINERS.md"
+  - "CONTRIBUTING.md"
+  - "CODE_OF_CONDUCT.md"
+
+gitIgnored: true
+localLinksStyle:
+  hugo:
+    indexFileName: "_index.md"
+
+transformations:
+
+  - glob: "../CHANGELOG.md"
+    path: /thanos/CHANGELOG.md
+    frontMatter:
+      template: |
+        title: Changelog
+        type: docs
+        menu: thanos
+        lastmod: "{{ .Origin.LastMod }}"
+
+  - glob: "../MAINTAINERS.md"
+    path: /thanos/MAINTAINERS.md
+    frontMatter:
+      template: |
+        title: Maintainers
+        type: docs
+        menu: thanos
+        lastmod: "{{ .Origin.LastMod }}"
+
+  - glob: "../SECURITY.md"
+    path: /thanos/SECURITY.md
+    frontMatter:
+      template: |
+        title: Security
+        type: docs
+        menu: thanos
+        lastmod: "{{ .Origin.LastMod }}"
+
+  - glob: "../CODE_OF_CONDUCT.md"
+    path: /contributing/CODE_OF_CONDUCT.md
+    frontMatter:
+      template: |
+        title: Code of Conduct
+        type: docs
+        menu: contributing
+        lastmod: "{{ .Origin.LastMod }}"
+
+  - glob: "../CONTRIBUTING.md"
+    path: /contributing/CONTRIBUTING.md
+    frontMatter:
+      template: |
+        title: Contributing
+        type: docs
+        menu: contributing
+        lastmod: "{{ .Origin.LastMod }}"
+  
+  - glob: "components/README.md"
+    path: /components/_index.md
+    frontMatter:
+      template: |
+        title: "{{ .Origin.FirstHeader }}"
+
+  - glob: "operating/README.md"
+    path: /operating/_index.md
+    frontMatter:
+      template: |
+        title: "{{ .Origin.FirstHeader }}"
+
+  - glob: "contributing/README.md"
+    path: /contributing/_index.md
+    frontMatter:
+      template: |
+        title: "{{ .Origin.FirstHeader }}"
```

I wonder if just one element will be enough:

```suggestion
  - glob: "**/README.md"
    path: _index.md
    frontMatter:
      template: |
        title: "{{ .Origin.FirstHeader }}"
```